bin/syclcc-clang

```diff
@@ -1125,6 +1174,59 @@ class omp_invocation:
 
     return linker_args
 
+class omp_accelerated_invocation:
+
+  @property
+  def unique_name(self):
+    return "omp.accelerated"
+
+  @property
+  def is_integrated_multipass(self):
+    return True
+
+  @property
+  def is_explicit_multipass(self):
+    return not self.is_integrated_multipass
+
+  def __init__(self, config, targets):
+    self._linker_args = config.omp_link_line
+    self._cxx_flags = config.omp_cxx_flags
+
+    if len(targets) != 0:
+      raise RuntimeError("OpenMP backend does not support specifiying target architecture")
+    if not config.has_plugin_cpu_acceleration:
+      raise RuntimeError("Trying to use accelerated CPU variant, but plugin was built without support for it")
+
+    self._clang_path = config.clang_path
+
+  def get_compiler_preference(self):
+    return (self._clang_path, 10)
+
+  def get_host_pass_requirements(self):
+    return {
+      'requires-extended-host-pass' : False,
+      'extended-host-pass-providers' : [],
+      'conflicts' : ["cuda-nvcxx"],
```

Comment:
```suggestion
      'conflicts' : ["cuda-nvcxx", "omp.library-only"],
```
Just noticed you can do `--hipsycl-targets="omp;omp.accelerated"` - this should probably be prevented, because we cannot actually have both simultaneously, right?

Reply:
good catch :)